docs/coordinates/matchsep.rst

```diff
@@ -250,7 +251,7 @@ the catalog:
 .. doctest-requires:: scipy
 
     >>> matches = catalog[idx]
-    >>> (matches.separation_3d(c) == d3d).all()
```

Comment:
this one is correct, I think

Reply:
I would wait to see what scipy end up doing if we going to pin in https://github.com/astropy/astropy/pull/16330 , just in case if their fix also get rid of the need to update this.